HCaptcha/Classes/HCaptchaWebViewManager.swift

```diff
@@ -385,4 +367,24 @@ extension HCaptchaWebViewManager: WKNavigationDelegate {
         }
         decisionHandler(WKNavigationActionPolicy.allow)
     }
+
+    func webView(_ webView: WKWebView, didFail navigation: WKNavigation!, withError error: Error) {
+        completion?(HCaptchaResult(error: .unexpected(error)))
+    }
+
+    func webView(_ webView: WKWebView, didFailProvisionalNavigation navigation: WKNavigation!, withError error: Error) {
+        completion?(HCaptchaResult(error: .unexpected(error)))
+    }
```

Comment:
This looks a little weird. Also, please add comments to these functions.

Reply:
To give more context, navigation happens each time when we load content in web view and I think we must catch those issue too as unexpected.

I will add comments